packages/webapp/package.json

```diff
@@ -12,6 +12,7 @@
     "@fluentui/react": "^8.110.7",
     "@fluentui/react-icons": "^2.0.206",
     "@react-spring/web": "^9.7.3",
+    "chat-component": "file:../chat-component",
```

Comment:
Must be installed the "regular" way within an NPM workspace, otherwise it may cause issues